contracts/dtoken/tests/userflows/userflow.repay.feature

```diff
@@ -0,0 +1,53 @@
+Feature: User Repay flow
+
+Background:
+	Given The user A
+	And Underlying token contract WNEAR with 100 tokens for user A
+	And Underlying token contract WETH with 100 tokens for user A
+	And Underlying token contract WBTC with 100 tokens for user A
+	And Digital token DWNEAR contract with supplied 70 tokens by user A
+	And Digital token DWETH contract with supplied 60 tokens by user A
+	And Digital token DWBTC contract with supplied 0 tokens by user A
+	And Digital token DWNEAR contract with borrows of 50 tokens by user A
+	And Digital token DWETH contract with borrows of 40 tokens by user A
+	And Digital token DWBTC contract with borrows of 0 tokens by user A
+	And Exchange_rate for contracts equal 1 
```

Comment:
@pahomas let's write it in other terms, we don't control exchange rate, we can control underlying balance, supplies, borrows and total reserve